index.Rmd

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Can you please put the `output` related metadata in `_output.yml`? I think this file should containing something more like this:

```yaml
knit: "bookdown::render_book"
title: "ggplot2: elegant graphics for data analysis"
author: ["Hadley Wickham"]
url: 'https\://r4ds.had.co.nz/'
github-repo: hadley/ggplot2-book
twitter-handle: hadley
site: bookdown::bookdown_site
documentclass: book
```